code/game/machinery/rechargestation.dm

```diff
@@ -289,3 +289,27 @@
 	if(!usr.incapacitated())
 		return
 	go_in(usr)
+
+/obj/machinery/recharge_station/ghost_pod_recharger
+	name = "drone pod"
+	desc = "This is a pod which used to contain a drone.. Or maybe it still does?"

```

Comment:
Should have three dots.

Reply:
It's grammatically correct to use three dots? Amazing.